pyaerocom/io/mscw_ctm/reader.py

```diff
@@ -56,6 +57,14 @@ class ReadMscwCtm(GriddedReader):
         data_dir must contain at least one of Base_(hour|day|month|fullrun).nc
         For multi-year analysis/trends, datadir may contain subdirectories named by trend-year,
         e.g. 2010 or trend2010.
+    file_pattern : str | re.Pattern, optional
```

Comment:
Do you want people to send you a precompiled re.Pattern? Why? A string which can be compiled as Pattern should be enough.